api/lib/source/polkadotV0-source.js

```diff
@@ -725,7 +725,10 @@ class polkadotAPI {
     return orderBy(allProposals, 'id', 'desc')
   }
 
-  getProposalById() {}
+  async getProposalById(proposalId) {
```

Comment:
yep, nice

Reply:
this is working now like it should. No need to query all proposals in the `PageProposal` component